### PR TITLE
Bugfix: Add keys missing on the return object of parsed certificates

### PR DESCRIPTION
**Pull request by joshuaharry**
Please fill in this template.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:
See https://github.com/encharm/x509.js/blob/84a46e220a067b8ae26f3b795bc62233029d4088/tests/parseCert.js#L48

### CODE REVIEW

File: types/x509.js/index.d.ts [hunk 1]
```diff
@@ -8,9 +8,16 @@ export interface Certificate {
     publicExponent: string;
     subject: {
         commonName: string;
+        organizationalUnitName: string;
```
Comment (Moser-ss) [inline types/x509.js/index.d.ts:11]: This should be optional, not every certificate have organizationalUnitName

File: types/x509.js/index.d.ts [hunk 2]
```diff
@@ -8,9 +8,16 @@ export interface Certificate {
...
     };
     issuer: {
         commonName: string;
+        countryName: string;
+        localityName: string;
```
Comment (Moser-ss) [inline types/x509.js/index.d.ts:16]: From the example I have this should be optional

File: types/x509.js/index.d.ts [hunk 3]
```diff
@@ -8,9 +8,16 @@ export interface Certificate {
...
+        organizationName: string;
+        organizationalUnitName: string;
+        serialNumber: string;
+        stateOrProvinceName: string;
```
Comment (Moser-ss) [inline types/x509.js/index.d.ts:20]: From the example I have this should be optional

**Review by Moser-ss — CHANGES_REQUESTED** [review]

**Review by Moser-ss — APPROVED** [review]
Looks ok

**Comment by joshuaharry** [comment]
Ready to merge

**Pull request merged**